sktime/annotation/ggs.py

```diff
@@ -34,19 +43,46 @@ class GGS:
     """
     Greedy Gaussian Segmentation.
 
+    The method apprixmates solutions for the problem of breaking a
+    multivariate time series into segments, where the data in each segment
+    could be modeled as independent samples from a multivariate Gaussian
+    distribution. It uses a dynamic programming search algorithm with
+    a heuristic that allows finding approximate solution in linear time with
+    respect to the data length and always yields locally optimal choice.
+
+    Greedy Gaussian Segmentation (GGS) fits a segmented gaussian model (SGM)
+    to the data by computing the appriximate solution to the combinatorial
+    problem of finding the approximate covariance-regularized  maximum
```

Comment:
I find it a bit confusing from a developer perspective that the docstring for the `GGS` class is the same as the one for `GreedyGaussianSegmentation` class below. I know this docstring isn't showing up on the website - so it isn't an issue there, but I think it would be helpful for anyone looking at your code in the future to be clued into the fact that `GGS` is the algorithmic implementation that is turned into an estimator in `GreedyGaussianSegmentation` here (just one or two sentences to give people a sense of the design would suffice).

Reply:
Added module level paragraph on the design.